SPECS/prebuilt-ca-certificates/prebuilt-ca-certificates.spec

```diff
@@ -0,0 +1,45 @@
+Summary:        Prebuilt version of ca-certificates package.
+Name:           prebuilt-ca-certificates
+Version:        20200720
+Release:        1%{?dist}
+License:        MIT
+Vendor:         Microsoft Corporation
+Distribution:   Mariner
+Group:          System Environment/Security
+URL:            https://hg.mozilla.org
+BuildArch:      noarch
+
+%description
+Prebuilt versions of the ca-certificates package with no runtime dependencies.
+
+%package base
```

Comment:
I though about packaging a bit more and I think we should just go ahead and call this whole package `prebuilt-ca-certificates-base` and make it provide only the default package. If we ever need another prebuilt package, we'd need to create a separate spec file.

Justification: if we ever wanted to build a different prebuilt certificates package, then that package would have a `BuildRequires` on the original cert bundle (`ca-certificiates-microsoft` for instance). We wouldn't be able to build it alongside the current `prebuilt-ca-certificates-base`, because `BuildRequires` are installed all at once and having both `ca-certificates-base` and `ca-certificates-microsoft` would still generate the same set of bundle files but now containing both the base and Microsoft certs. These bundle files are exactly what the prebuilt packages need to contain, so we'd end up with both prebuilt packages installing the same set of certificates. In both cases that set would be too large.

Reply:
fair point